test_app/lib/screens/participant_screen/participant_screen.dart

```diff
@@ -249,4 +265,47 @@ class _ParticipantScreenContentState extends State<ParticipantScreenContent> {
     }
     return Future.value();
   }
+
+  Future<void> _updateShareScreenView() async {
+    final participants = await _dolbyioCommsSdkFlutterPlugin.conference
+        .getParticipants(widget.conference);
+    final availableParticipants = participants
+        .where((element) => element.status != ParticipantStatus.left);
+
+    MediaStream? stream;
+    Participant? streamingParticipant;
+    bool isScreenSharing = false;
+
+    if (availableParticipants.isNotEmpty) {
+      for (var participant in availableParticipants) {
+        var streams = participant.streams;
+        if (streams != null) {
+          for (var s in streams) {
```

Comment:
I think the code readability would be easy if the variable is named instead of "s".